htdocs/core/lib/functions.lib.php

```diff
@@ -6380,8 +6380,14 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 			$substitutionarray['__REF_SUPPLIER__'] = (isset($object->ref_supplier) ? $object->ref_supplier : null);
 			$substitutionarray['__NOTE_PUBLIC__'] = (isset($object->note_public) ? $object->note_public : null);
 			$substitutionarray['__NOTE_PRIVATE__'] = (isset($object->note_private) ? $object->note_private : null);
-
 			$substitutionarray['__DATE_DELIVERY__'] = (isset($object->date_livraison) ? dol_print_date($object->date_livraison, 'day', 0, $outputlangs) : '');
+			$datedelivery = dol_getdate($object->date_livraison, true);
+			$substitutionarray['__DATE_DELIVERY_DAY__'] = (isset($object->date_livraison) ? sprintf("%02d", $datedelivery['mday']) : '');
```

Comment:
To get the day, hour, month, etc of a date, use instead   
dol_print_date("%d", $object->date_livraison)
dol_print_date("%H", $object->date_livraison)
...

Reply:
@eldy I have update

Reply:
We've got error in CI due to syntax of code. No space found after comma in argument list

Reply:
@eldy I have update